ui/lib/kv/addon/components/page/secret/metadata/details.js

```diff
@@ -54,4 +59,22 @@ export default class KvSecretMetadataDetails extends Component {
       this.flashMessages.danger(`There was an issue deleting ${path} metadata. \n ${errorMessage(err)}`);
     }
   }
+
+  @action
+  async requestData() {
+    const { backend, path } = this.args;
+    try {
+      const secretData = await this.store.queryRecord('kv/data', { backend, path });
+      this.customMetadataFromData = secretData.customMetadata;
+    } catch (error) {
+      if (error.message === 'Control Group encountered') {
+        this.controlGroup.saveTokenFromError(error);
+        this.error = this.controlGroup.logFromError(error);
+        this.error.isControlGroup = true;
+        return;
+      }
+      this.error.isControlGroup = false;
+      this.error = errorMessage(error);
```

Comment:
This displays using the `MessageError` component
<img width="776" alt="Screenshot 2024-08-30 at 10 33 42 AM" src="https://github.com/user-attachments/assets/da9fd633-fe4d-499c-9cae-89b5173b7c3c">